packages/mwp-app-render/src/components/uxcapture/UXCaptureFont.jsx

```diff
@@ -0,0 +1,44 @@
+S.prototype.g=function(a){var b=this.a;if(b.f){var c=x(b.c,b.a.b("wf",a.b,E(a).toString(),"active")),d=[],e=[b.a.b("wf",a.b,E(a).toString(),"loading")];c||d.push(b.a.b("wf",a.b,E(a).toString(),"inactive"));w(b.c,d,e)}F(b,"fontinactive",a);U(this)};function U(a){0==--a.c&&a.h&&(a.i?(a=a.a,a.f&&w(a.c,[a.a.b("wf","active")],[a.a.b("wf","loading"),a.a.b("wf","inactive")]),F(a,"active")):G(a.a))};function V(a){this.h=a;this.f=new ka;this.g=0;this.a=this.c=!0}V.prototype.load=function(a){this.b=new ba(this.h,a.context||this.h);this.c=!1!==a.events;this.a=!1!==a.classes;na(this,new ia(this.b,a),a)};
+function oa(a,b,c,d,e){var f=0==--a.g;(a.a||a.c)&&setTimeout(function(){var a=e||null,m=d||null||{};if(0===c.length&&f)G(b.a);else{b.c+=c.length;f&&(b.h=f);var k,n=[];for(k=0;k<c.length;k++){var C=c[k],X=m[C.b],p=b.a,u=C;p.f&&w(p.c,[p.a.b("wf",u.b,E(u).toString(),"loading")]);F(p,"fontloading",u);p=null;if(null===T)if(window.FontFace){var u=/Gecko.*Firefox\\/(\\d+)/.exec(window.navigator.userAgent),pa=/OS X.*Version\\/10\\..*Safari/.exec(window.navigator.userAgent)&&/Apple/.exec(window.navigator.vendor);
+T=u?42<parseInt(u[1],10):pa?!1:!0}else T=!1;T?p=new K(l(b.f,b),l(b.g,b),b.b,C,b.j,X):p=new L(l(b.f,b),l(b.g,b),b.b,C,b.j,a,X);n.push(p)}for(k=0;k<n.length;k++)n[k].start()}},0)}function na(a,b,c){var d=[],e=c.timeout;ja(b);var d=la(a.f,c,a.b),f=new S(a.b,b,e);a.g=d.length;b=0;for(c=d.length;b<c;b++)d[b].load(function(b,c,d){oa(a,f,b,c,d)})};function W(a,b){this.b=a;this.a=b}W.prototype.load=function(a){var b,c,d=this.a.urls||[],e=this.a.families||[],f=this.a.testStrings||{},g=new ea;b=0;for(c=d.length;b<c;b++)da(this.b,d[b],fa(g));var m=[];b=0;for(c=e.length;b<c;b++)if(d=e[b].split(":"),d[1])for(var k=d[1].split(","),n=0;n<k.length;n+=1)m.push(new A(d[0],k[n]));else m.push(new A(d[0]));ga(g,function(){a(m,f)})};var Y=new V(window);Y.f.b.custom=function(a,b){return new W(b,a)};var Z={load:l(Y.load,Y)};"function"===typeof define&&define.amd?define(function(){return Z}):"undefined"!==typeof module&&module.exports?module.exports=Z:(window.WebFont=Z,window.WebFontConfig&&Y.load(window.WebFontConfig));}());`;
+
+const generateUXCaptureFontJS = (fontFamily: string, mark: string) => `
+    WebFont.load({
+        custom: {
+            families: ['${fontFamily}']
+        },
+        active: function() {
+            if(window.UXCapture) {
+                window.UXCapture.mark("${mark}");
+            }
+        }
+    });
+`;
+
+// fontFamily attribute should include individual weights if separate files are used
+// Example: "Graphik Meetup:n4,n5,n6"
+// Attention, if no weight specified, only n4 (e.g. font-weight: 400) is tested for!
+// See https://github.com/typekit/webfontloader#events for more detail
+const UXCaptureFont = ({ fontFamily, mark }: Props) =>
+	<script
+		dangerouslySetInnerHTML={{
```

Comment:
Are you sure this works? I don't think it will. This is why we were having problems when pairing. See here: https://github.com/facebook/react/issues/8838#issuecomment-314740580
```
script tags injected via innerHTML are not executed for security reasons:

https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML

Although this may look like a cross-site scripting attack, the result is harmless. HTML5 specifies that a <script> tag inserted via innerHTML should not execute.
```